docs/presets/README.md

```diff
@@ -89,6 +89,9 @@ symbol = " "
 [shlvl]
 symbol = " "
 
+[spack]
+symbol = " "
```

Comment:
This (and other presets) will need to be rebased onto master, since 1.4.0 brought some changes to the preset system.

The new TOML files are in `docs/.vuepress/public/presets/toml`--updating those should take care of everything.